### PR TITLE
Take advantage of multi files COPY to simplify Dockerfile

### PR DESCRIPTION
**Pull request by elfosardo**
Move scripts under scripts dir, then copy everything at the same
time when possible.
Also no need to specify ./ with COPY, the default path is always
in the source dir.

### CODE REVIEW

**Comment by zaneb** [comment]
Need to rename `scripts/runironic-inspector.sh` to `scripts/runironic-inspector`.

File: Dockerfile [hunk 1]
```diff
@@ -2,17 +2,14 @@ FROM docker.io/centos:centos8
 
 ARG PKGS_LIST=main-packages-list.txt
 
-COPY ${PKGS_LIST} /tmp/main-packages-list.txt
+COPY ${PKGS_LIST} /tmp/
```
Comment (zaneb) [inline Dockerfile]: If the user supplies a different arg for PKGS_LIST we still want it to end up at `/tmp/main-packages-list.txt`, so I don't think this change is correct.

Comment (elfosardo) [reply Dockerfile]: deja-vu :)

**Comment by elfosardo** [comment]
> Need to rename `scripts/runironic-inspector.sh` to `scripts/runironic-inspector`.

done

**Review by bfournie — APPROVED** [review]

**Comment by elfosardo** [comment]
@zaneb hi! Can you please have another look? Thanks!

File: Dockerfile [hunk 2]
```diff
@@ -8,11 +8,8 @@ COPY prepare-image.sh /bin/
 RUN prepare-image.sh && \
   rm -f /bin/prepare-image.sh
 
-COPY ./inspector.conf.j2 /etc/ironic-inspector/ironic-inspector.conf.j2
-COPY ./runironic-inspector.sh /bin/runironic-inspector
-COPY ./runhealthcheck.sh /bin/runhealthcheck
-COPY ./ironic-common.sh /bin/ironic-common.sh
-COPY ./runlogwatch.sh /bin/runlogwatch.sh
+COPY inspector.conf.j2 /etc/ironic-inspector/
```
Comment (zaneb) [inline Dockerfile]: Renaming `ironic-inspector.conf.j2` to `inspector.conf.j2` without changing [the script that refers to it](https://github.com/metal3-io/ironic-inspector-image/blob/master/runironic-inspector.sh#L3) will break.

Comment (elfosardo) [reply Dockerfile]: thanks, fixed

**Review by bfournie — APPROVED** [review]

**Pull request merged**